src/dotnet/commands/dotnet-msbuild/MSBuildForwardingLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+
+namespace Microsoft.DotNet.Tools.MSBuild
+{
+    public sealed class MSBuildForwardingLogger : IForwardingLogger
+    {
+        public LoggerVerbosity Verbosity { get; set; } = LoggerVerbosity.Minimal;
```

Comment:
Why do we need to set a default here?

Reply:
I guess not technically needed, but better to be explicit? Default looks like it might be Quiet. But I don't know that this has any effect unless it's diagnostic (we have code that checks for that).

Reply:
I guess I don't understand the significance of this member. I thought it was input from the engine to the logger and would get set to whatever verbosity is being logged. We don't care about verbosity in this logger. We always want to get telemetry events. What changes if this defaults to diagnostic?